mdx-models/src/main/java/com/mx/path/model/mdx/accessor/account/AccountBaseAccessor.java

```diff
@@ -30,7 +30,7 @@ public abstract class AccountBaseAccessor extends Accessor {
 
   @GatewayAPI
   @Getter(AccessLevel.PROTECTED)
-  private AccountNumberBaseAccessor accountNumbers;
+  private AccountNumbersBaseAccessor accountNumbers;
```

Comment:
This was bugging me, I'm applaud this change haha

Reply:
🧹🧹🧹